content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -402,7 +402,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `KubeletPluginsWatcher`: Enable probe-based plugin watcher utility to enable kubelet
   to discover plugins such as [CSI volume drivers](/docs/concepts/storage/volumes/#csi).
 - `KubeletPodResources`: Enable the kubelet's pod resources grpc endpoint.
-   See [Support Device Monitoring](https://git.k8s.io/community/keps/sig-node/compute-device-assignment.md) for more details.
+   See [Support Device Monitoring](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/compute-device-assignment.md) for more details.
```

Comment:
```suggestion
   See the [kubelet endpoint for device assignment observation](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/compute-device-assignment.md) KEP for more details.
```
(copying the title from the target; it changed since the KEP was drafted).